src/PageLayout/PageLayout.stories.tsx

```diff
@@ -5,7 +5,7 @@ import {Placeholder} from '../Placeholder'
 import {PageLayout} from './PageLayout'
 
 const meta: Meta = {
-  title: 'Layout/PageLayout',
+  title: 'Components/PageLayout',
```

Comment:
nit, not blocking: do you think we need another layer of nesting? Components/Layouts/PageLayout?